src/Rasterization/Renderers/SVGRenderer.php

```diff
@@ -57,6 +50,13 @@ public function render(SVGRasterizer $rasterizer, array $options, SVGNode $conte
 
             $this->renderStroke($image, $params, $stroke, $strokeWidth);
         }
+
+        $fill = $context->getComputedStyle('fill');
+        if (isset($fill) && $fill !== 'none') {
+            $fill = self::prepareColor($fill, $context);
+
+            $this->renderFill($image, $params, $fill);
+        }
```

Comment:
To cite the SVG 2 specification, § 3.7.1, third paragraph:

> The order in which fill, stroke and markers are painted is determined by the paint-order property. The default is that fill is painted first, then the stroke, and then the marker symbols. [...]

As such, changing the paint order here is incorrect.

This also means the "renderer-rect-stroke-fill.png" test image should not be replaced.

Reply:
Of course, I understand that doing it this way is required for rendering text with `imagettftext`. We need to come up with a solution to that.